SYCL/ESIMD/imulh_umulh.cpp

```diff
@@ -10,6 +10,9 @@
 // RUN: %clangxx -fsycl %s -o %t.out
 // RUN: %GPU_RUN_PLACEHOLDER %t.out
 
+// TODO: the test gives irregular fails in CI (flaky), disabled until resolved.
+// XFAIL: *
```

Comment:
I think the right way to disable flaky fails is to use `UNSUPPORTED`. XFAIL expects test to fail **always** i.e. you will get unexpected pass. That's what's happening right now in CI. I see a lot job fail due to unexpected pass.
@v-klochkov, please, fix ASAP.